shells/lib/elements/launcher-arc.js

```diff
@@ -27,7 +27,7 @@ const log = logFactory('LauncherArc', '#cb23a6');
 const LauncherArcElement = class extends ArcElement {
   // get config() {
```

Comment:
commented code here too.

Reply:
I need to decide which direction to go with this legacy code, but I'm not ready to make that decision yet, so I left it as-is (modulo the namespace path I updated to be proactive).